ui/src/components/profile/Profile.js

```diff
@@ -0,0 +1,64 @@
+import React, {useEffect, useCallback, useState} from 'react';
+import {useHttp} from '../../hooks/http.hook';
+
+import PropTypes from 'prop-types';
+
+import ShipperProfile from './ShipperProfile/ShipperProfile';
+import DriverProfile from './DriverProfile/DriverProfile';
+
+
+const Profile = props => {
+
+  const { request } = useHttp();
+
+  const storageName = 'userData';
+
+  const [user, setUser] = useState({
+    id: '',
+    name: '',
+    email: '',
+    role: '',
+  });
+
+
+  useEffect( useCallback(() => {
+    async function fetchData() {
+      try {
+        const storeData = JSON.parse(localStorage.getItem(storageName));
+
+        const user = await request(`/profile/${storeData.userId}`, 'GET', null, {
+          'Content-Type': 'application/json',
+          'Authorization': storeData.token,
+          'Role': storeData.role,
```

Comment:
its not so secure, because any one can send request and add header with role, can we take it from our token on backend side ?